nextcord/application_command.py

```diff
@@ -2985,8 +2988,10 @@ def message_command(
     name_localizations: Dict[Union[:class:`Locale`, :class:`str`], :class:`str`]
         Name(s) of the command for users of specific locales. The locale code should be the key, with the localized
         name as the value
-    guild_ids: Iterable[:class:`int`]
-        IDs of :class:`Guild`'s to add this command to. If unset, this will be a global command.
+    guild_ids: Optional[Iterable[:class:`int`]]
+        IDs of :class:`Guild`'s to add this command to. If set to :attr:`utils.MISSING` and :attr:`Client.default_guild_ids` is
```

Comment:
`MISSING` isnt technically documented right now, and its not an attribute but simply a `:data:` (non-class/function/method/...), so probably best to just say "if not passed"

Reply:
I think this is what should be used everywhere MISSING is referenced in docstrings in this PR. That would be consistent with what has usually been done.